.github/workflows/build.yaml

```diff
@@ -1,10 +1,12 @@
 name: Build
 on: [push]
+env:
+  DEVELOPER_DIR: /Applications/Xcode_14.1.app/Contents/Developer
 jobs:
   build:
     strategy:
       matrix:
-        os: [windows-2022, macos-11, ubuntu-22.04]
+        os: [windows-2022, macos-12, ubuntu-22.04]
```

Comment:
this didn't end up being needed for fixing it, should I revert it?

Reply:
Sure, let's try to keep all our builds on the same version.

Reply:
(apparently hasn't yet happened)